tests/xtro-sharpie/SelectorCheck.cs

```diff
@@ -51,24 +34,49 @@ public override void VisitManagedMethod (MethodDefinition method)
 			foreach (var ca in method.CustomAttributes) {
 				switch (ca.Constructor.DeclaringType.Name) {
 				case "ExportAttribute":
-					string selector = ca.ConstructorArguments [0].Value as string;
-					if (!known_selectors.Contains (selector))
-						known_selectors.Add (selector);
+					var methodDefinition = method.GetName ();
+					if (!string.IsNullOrEmpty (methodDefinition)) {
+						var argumentSemantic = Helpers.ArgumentSemantic.None;
```

Comment:
`None` is not a value - it means the default (pre-ARC it's `Assign`)
The comparison you make later might not be correct.

Reply:
Here's an example
```
!incorrect-argument-semantic! 'CNContainer::type' (native) has (none) instead of 'ArgumentSemantic.UnsafeUnretained|Assign'
```
and the default (when nothing is specified) is `assign` - that's a false positive.